cmd/monaco/deploy/command.go

```diff
@@ -47,12 +47,12 @@ func GetDeployCommand(fs afero.Fs) (deployCmd *cobra.Command) {
 				return err
 			}
 
-			return deployConfigs(fs, manifestName, group, environment, project, continueOnError, dryRun)
+			return deployConfigs(fs, manifestName, groups, environment, project, continueOnError, dryRun)
 		},
 	}
 
-	deployCmd.Flags().StringSliceVarP(&environment, "environment", "e", make([]string, 0), "Specify one (or multiple) environments to deploy to. To set multiple environments either repeat this flag, or seperate them using a comma (,). This flag is mutually exclusive with '--group'.")
-	deployCmd.Flags().StringVarP(&group, "group", "g", "", "Specify the environmentGroup that should be used for deployment. If this flag is specified, all environments within this group will be used for deployment. This flag is mutually exclusive with '--environment'")
+	deployCmd.Flags().StringSliceVarP(&environment, "environment", "e", []string{}, "Specify one (or multiple) environment(s) to deploy to. To set multiple environments either repeat this flag, or separate them using a comma (,). This flag is mutually exclusive with '--group'.")
+	deployCmd.Flags().StringSliceVarP(&groups, "group", "g", []string{}, "Specify one (or multiple) environmentGroup(s) to deploy to. To set multiple groups either repeat this flag, or seprate them using a comma (,). If this flag is specified, all environments within this group will be used for deployment. This flag is mutually exclusive with '--environment'")
```

Comment:
did we already wrongly describe this as repeatable in the documentation, or will we have to update it there as well?

Reply:
Currently, I would argue that the docs are wrong: `Apply your configurations to specific environment groups within your deployment manifest file.` (groups!)
With this change, the behavior is as the docs state it. Not sure on what level we want to keep the docs and Monaco-command-descriptions aligned. 

https://www.dynatrace.com/support/help/manage/configuration-as-code/reference/commands

Reply:
I think I probably documented that wrongly thinking that groups are like environments and copying the description. 

As for alignment, I'd say "generally correct" doesn't need to match wording but shouldn't be wrong and shouldn't be missing new commands/flags

Reply:
👍